pkg/dashboard/index.html

```diff
@@ -84,9 +84,9 @@
                 <div class="panel panel-default" id="dashboard-hosts">
                     <div class="panel-heading">
                         <button class="btn btn-primary servers-privileged fa fa-plus"
-                            id="dashboard-add" title="Add Server"></button>
+                            id="dashboard-add" aria-label="Add Server" title="Add Server"></button>
```

Comment:
This isn't translated currently.

I guess the `aria-label` and `title` should be added by JavaScript? (It's usually better to use `aria-labelledby` instead, and point it at the ID of something with the text, which usually can be translated.)

This was untranslated before I got to this; I just duplicated the issue in the more accessible version of the element.